processor/tailsamplingprocessor/README.md

```diff
@@ -108,6 +109,11 @@ processors:
              type: trace_state,
              trace_state: { key: key3, values: [value1, value2] }
          },
+         {
+              name: test-policy-12,
+              type: boolean_attribute,
+              boolean_attrbiute: {key: key4, value: true}
```

Comment:
Could you replace `key4` with a realistic example? I know other examples aren't realistic, but it would be clearer if this would list something people would use in the real world.